__tests__/app.test.js

```diff
@@ -64,8 +80,44 @@ describe("GET /api/articles", () => {
         });
       });
   });
+  test("GET 200 - valid topic query should return articles matching topic", () => {
+    return request(app)
+      .get("/api/articles?topic=cats")
+      .expect(200)
```

Comment:
thanks, yep I can check for empty topic. I was intending to add functionality to dynamically update the reference whenever a new topic or user was added or removed, but since I'm not near a point in my project where that will happen, I'll refactor to use a db check function in utils instead.